shared/src/business/useCases/serveCaseToIrs/generateDraftDocument.test.js

```diff
@@ -0,0 +1,81 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const {
+  formatDateString,
+  formatNow,
+  FORMATS,
+  getBusinessDateInFuture,
+} = require('../../utilities/DateHandler');
+const {
+  INITIAL_DOCUMENT_TYPES,
+  SYSTEM_GENERATED_DOCUMENT_TYPES,
+} = require('../../entities/EntityConstants');
+const { Case } = require('../../entities/cases/Case');
+const { generateDraftDocument } = require('./generateDraftDocument');
+const { MOCK_CASE } = require('../../../test/mockCase');
+
+describe('generateDraftDocument', () => {
```

Comment:
@rachelschneiderman, here's the new test file.

Reply:
Oh ok! I thought we agreed yesterday to try the sociable testing strategy instead? What changed your mind?

Reply:
I'm honestly confused. I thought we agreed to talk about trying that out, but if you're okay with it here, I can just revert that commit. Honestly the test basically does nothing. There's a single path through the code, so we're validating that the `replaceBracketed` function works because it would make absolutely no sense to mock that, because the only other thing is to call a function that I've now mocked with the result.

Reply:
To be clear, the existing tests are very sociable. We're just covering a lot of code several times—from different test files—it would seem.